src/h2n_fetcher.erl

```diff
@@ -0,0 +1,396 @@
+parse_license1(_Name, <<"efl 2.0">>) -> <<"efl20">>;
+parse_license1(_Name, <<"efl-1.0">>) -> <<"efl10">>;
+parse_license1(_Name, <<"efl 1.0">>) -> <<"efl10">>;
+parse_license1(_Name, <<"cpl-1.0">>) -> <<"cpl10">>;
+parse_license1(_Name, <<"cpl 1.0">>) -> <<"cpl10">>;
+parse_license1(_Name, <<"cecill-c">>) -> <<"cecill-c">>;
+parse_license1(_Name, <<"cecill c">>) -> <<"cecill-c">>;
+parse_license1(_Name, <<"cecill-2.0">>) -> <<"cecill20">>;
+parse_license1(_Name, <<"cecill 2.0">>) -> <<"cecill20">>;
+parse_license1(_Name, <<"cddl">>) -> <<"cddl">>;
+parse_license1(_Name, <<"cddl-1.0">>) -> <<"cddl">>;
+parse_license1(_Name, <<"cddl 1.0">>) -> <<"cddl">>;
+parse_license1(_Name, <<"cc-by-sa-4.0">>) -> <<"cc-by-sa-40">>;
+parse_license1(_Name, <<"cc by sa 4.0">>) -> <<"cc-by--sa 40">>;
+parse_license1(_Name, <<"cc-by-4.0">>) -> <<"cc-by-40">>;
+parse_license1(_Name, <<"cc by 4.0">>) -> <<"cc-by-40">>;
+parse_license1(_Name, <<"cc-by-sa-3.0">>) -> <<"cc-by-sa-30">>;
+parse_license1(_Name, <<"cc-by-3.0">>) -> <<"cc-by-30">>;
+parse_license1(_Name, <<"cc by 3.0">>) -> <<"cc-by-30">>;
+parse_license1(_Name, <<"cc-by-sa-2.5">>) -> <<"cc-by-sa-25">>;
+parse_license1(_Name, <<"cc by sa 2.5">>) -> <<"cc-by-sa-25">>;
+parse_license1(_Name, <<"cc0-1.0">>) -> <<"cc0">>;
+parse_license1(_Name, <<"cc0 1.0">>) -> <<"cc0">>;
+parse_license1(_Name, <<"bsd-3">>) -> <<"bsd3">>;
+parse_license1(_Name, <<"bsd 3">>) -> <<"bsd3">>;
+parse_license1(_Name, <<"bsd-2">>) -> <<"bsd2">>;
+parse_license1(_Name, <<"bsd 2">>) -> <<"bsd2">>;
+parse_license1(_Name, <<"bsl-1.0">>) -> <<"boost">>;
+parse_license1(_Name, <<"bsl 1.0">>) -> <<"boost">>;
+parse_license1(_Name, <<"apache-2.0">>) -> <<"asl20">>;
+parse_license1(_Name, <<"apache 2.0">>) -> <<"asl20">>;
+parse_license1(_Name, <<"artistic-1.0">>) -> <<"artistic1">>;
+parse_license1(_Name, <<"artistic 1.0">>) -> <<"artistic1">>;
+parse_license1(_Name, <<"artistic">>) -> <<"artistic1">>;
+parse_license1(_Name, <<"apsl-2.0">>) -> <<"apsl20">>;
+parse_license1(_Name, <<"apsl 2.0">>) -> <<"apsl20">>;
+parse_license1(_Name, <<"agpl-3.0">>) -> <<"agpl3">>;
+parse_license1(_Name, <<"agpl 3.0">>) -> <<"agpl3">>;
+parse_license1(_Name, <<"afl-2.1">>) -> <<"afl21">>;
+parse_license1(_Name, <<"afl 2.1">>) -> <<"afl21">>;
+parse_license1(_Name, <<"apache 2">>) -> <<"apsl20">>;
+parse_license1(_Name, <<"unlicense">>) -> <<"unlicense">>;
+parse_license1(_Name, <<"mozilla public license 1.1">>) -> <<"mpl11">>;
+parse_license1(_Name, <<"isc">>) -> <<"isc">>;
+parse_license1(_Name, <<"the mit License (mit)">>) -> <<"mit">>;
+parse_license1(_Name, <<"mit License">>) -> <<"mit">>;
+parse_license1(_Name, <<"gpl (for code)">>) -> <<"gpl3">>;
+parse_license1(_Name, <<"bsd (for code)">>) -> <<"bsd3">>;
+parse_license1(_Name, <<"mit">>) -> <<"mit">>;
+parse_license1(_Name, <<"bsd">>) -> <<"bsd3">>;
+parse_license1(_Name, <<"wtfpl">>) -> <<"wtfpl">>;
+parse_license1(_Name, <<"the mit license">>) -> <<"mit">>;
+parse_license1(Name, License) ->
```

Comment:
I am open to that, but by definition everything in Hex is open, so its accurate. It could be the right thing to do is set the license and then in the generation write the TODO. So that it builds but still has the TODO if someone wants to see.